src/main/java/seedu/address/logic/commands/contact/EditCommand.java

```diff
@@ -7,7 +7,9 @@
 import static seedu.address.logic.parser.CliSyntax.PREFIX_CONTACT_PHONE;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_CONTACT_REMARK;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
-import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+//import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
```

Comment:
Could you remove the unused import?

Reply:
Whoops missed out on commented unused imports. Will remove them asap!